third-party/gasnet/Makefile

```diff
@@ -38,7 +38,7 @@ ifeq ($(CHPL_MAKE_TARGET_PLATFORM),pwr6)
 CHPL_GASNET_CFG_OPTIONS += -with-mpi-cc=mpcc
 endif
 
-ifneq (, $(filter $(CHPL_MAKE_TARGET_PLATFORM),cray-xe cray-xc cray-xk))
+ifneq (, $(filter $(CHPL_MAKE_TARGET_PLATFORM),cray-xe cray-xc cray-xk cross-aarch64))
```

Comment:
I'd leave aarch64 out of this conditional, and set CHPL_GASNET_CFG_SCRIPT (and --enable-pthreads if that's really necessary) in a separate check after just for aarch64.